user/noto/Quicklook.rst

```diff
@@ -7,59 +7,31 @@ Data formats and online quick-look
 Details on the FITS file structure are given in a separate document. Download 
 it here: :download:`pdf <attachments/MED-MAN-FITS-02.pdf>` 
 
-Waiting for a comprehensive GUI which is going to include also a real-time 
-preview of the data under acquisition, users are provided with two different 
+Users are provided with two different 
 tools in order to inspect the data produced by the TPB. 
 
 If writer is MANAGEMENT/FitsZilla
 =================================
-When acquiring FITS files through a schedule, there is an IDL tool available 
+When acquiring FITS files through a schedule, there is a tool available 
 for the almost-realtime quick-look of the saved data.
  
-Open a terminal on *euser*. Launch IDL::
+Open an internet browser and visit the page "http://XXX.XXX.XXX.XXX:8080". The address
```

Comment:
correspond to discos-manager workstation. Otherwise if you're directly --> 
corresponds to the discos-manager workstation. Otherwise if you are directly